### ISSUE TITLE
keybase.io link is dead

### ISSUE THREAD

**Issue by laf0rge** [rustyrussell/bitcoin-storage-guide#8]
The readme contains a link to https://keybase.io/rusty which seems dead by now.

**Comment by GlenCooper** [comment]
Agreed.  It's dead.  I went to open an issue on it, saw this already here, so I'm just reaffirming it's broken.
![Screenshot from 2020-08-02 03-49-55](https://user-images.githubusercontent.com/967793/89115104-c171a700-d473-11ea-9f94-154c3e937966.png)

![Screenshot from 2020-08-02 03-50-21](https://user-images.githubusercontent.com/967793/89115108-c898b500-d473-11ea-8372-83f05c4a7172.png)
